Testreglar/2.2.2/App/app-2.2.2a.json

```diff
@@ -0,0 +1,212 @@
+			"spm": "Har appsida innhald som automatisk beveger seg, blinkar eller rullar?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjekk om det finst innhald på appsida som automatisk beveger seg, blinkar eller rullar.</p>\n<p>Eksempel på denne typen innhald er video, animasjonar, spel og rullande børsnoteringar.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appside har ikkje innhald som automatisk beveger seg, blinkar eller rullar."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Kva innhald testar du?",
+			"ht": "<p>Beskriv innhaldet som beveger seg, blinkar eller rullar, slik at det er mogleg å identifisere det i ettertid.</p>\n<p>Viss det gjeld fleire element, registrerer du eit og eit.</p>",
+			"type": "tekst",
+			"label": "Innhald som beveger seg, blinkar, rullar:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Varer bevegelsen, blinkinga eller rullinga i meir enn 5 sekund?",
+			"ht": "<p>Bruk stoppeklokke eller lignande for å ta tida. </p>\n<p><strong>Merk:</strong> Video eller anna innhald som beveger seg, blinkar eller rullar, men varar kortere enn 5 sekund, er likevel omfatta viss det går i loop.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G11"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Innhald beveger seg, blinkar eller rullar i mindre enn 5 sekund."
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Viser bevegelsen, blinkinga eller rullinga samtidig med anna innhald?",
+			"ht": "<p><strong>Merk: </strong>Kravet gjeld etter ordlyden berre for innhald som beveger seg, blinkar eller rullar samtidig som anna innhald som blir vist på appsida.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Innhald beveger seg, blinkar eller rullar, utan å bli vist samtidig med anna innhald."
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er bevegelsen, blinkinga eller rullinga del av ei handling som er avgjerande?",
+			"ht": "<p>Avgjerande (nødvendige) handlingar er typisk noko som skjer i sanntid, og kor det å pause/stoppe/skjule vil gjere handlinga ugyldig.</p>\n<p>Eksempel kan vere handel på børs, aksjon, klokke og liknande situasjonar der brukaren treng live-oppdateringar.</p>\n<p><strong>Merk:</strong> Merknad 4 til suksesskriteriet lyder slik: \"En animasjon som inngår i forhåndsinnlasting eller lignende, kan anses som avgjørende dersom det ikke kan forekomme interaksjon for noen brukere i denne fasen, og dersom mangel på en framdriftsindikator kunne forvirre brukerne og skape inntrykk av at innholdet er enten frosset eller ødelagt.\"</p>",
```

Comment:
En animasjon for forhånds innlasting er nødvendig hvis dette er eneste framdriftsindikator.